### PR TITLE
feat: finemapping template and DAG for UKB PPP

### PR DESCRIPTION
**Pull request by tskir**
The idea is to have a common finemapping _template,_ which specific DAGs can reuse and modify according to their needs.

### CODE REVIEW

**Review by project-defiant — APPROVED** [review]

**Comment by tskir** [comment]
@project-defiant This has been quite substantially rewritten compared to the first draft. Could you do another round of reviews, please?

File: src/ot_orchestration/dags/ukb_ppp_finemapping.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+
+with DAG(
+    dag_id=Path(__file__).stem,
+    description="Open Targets Genetics — finemap study loci with SuSie",
+    default_args=common.shared_dag_args,
+    **common.shared_dag_kwargs,
+) as dag:
+    (
+        FinemappingBatchOperator.partial(
```
Comment (project-defiant) [inline src/ot_orchestration/dags/ukb_ppp_finemapping.py:41]: The partial will not work beyond the threshold, and I have tested it on local airflow DAG, this breaks on around ~5k partial tasks even with the threshold increase.

https://airflow.apache.org/docs/apache-airflow/stable/authoring-and-scheduling/dynamic-task-mapping.html#placing-limits-on-mapped-tasks

**Review by project-defiant — APPROVED** [review]
I think it is a good setup for me to use it during the gwas_catalog etl step (also for other ones that require finemapping).

File: src/ot_orchestration/dags/ukb_ppp_finemapping.py (hunk 1 above)
Comment (tskir) [reply src/ot_orchestration/dags/ukb_ppp_finemapping.py:41]: Indeed, but just to clarify, in this PR the _partial/expand_ routine iterates not on individual loci (of which there are potentially 100,000s in the worst case), but on chunks of the manifest, of which there are <10 in either case

**Pull request merged**